kube/int/create.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 # kubectl create -f dax-imgserv-datasets-volume.yaml
 # kubectl create -f dax-imgserv-datasets-claim.yaml --namespace $DAX_NAMESPACE
```

Comment:
Follow my pattern that you reviewed yesterday to create the volumes in another script.  Also, fix the names.

Reply:
Done